pybamm/input/parameters/lithium-ion/cathodes/LFP_Prada2013/parameters.csv

```diff
@@ -0,0 +1,35 @@
+Name [units],Value,Reference,Notes
+# Empty rows and rows starting with �#� will be ignored,,,

```

Comment:
Minor point: the characters around `#` look like that � (at least to me). In other `parameters.csv` they looked fine.